docs/GettingStarted/QuickstartTutorial.md

```diff
@@ -1,280 +1,366 @@
+$ cd mig-demo-apps/apps/todolist-mongo-go
+$ sed -i "s/your_org/${GITHUB_FORK_ORG}/g" mongo-persistent.yaml
+$ oc create -f mongo-persistent.yaml
+```
 
-### View the Pelorus measurements
+The todolist application and mongo database should now build and deploy into the `mongo-persistent` namespace.
 
-In your OpenShift Pelorus project page, open the link to Grafana or get the link, by running
+Wait for the application, by waiting for the ready state of the `pod` which is labeled `app=mongo`:
+```shell
+$ oc wait --for=condition=ready pod -l app=mongo --namespace mongo-persistent
 ```
-oc get route grafana-route --namespace pelorus -o=go-template='https://{{.spec.host | printf "%s\n" }}'
+
+Ensure that your BuildConfig points correctly to your **`URI value`** from your GitHub fork, by running:
+```shell
+$ oc get buildconfig.build.openshift.io/todolist --namespace mongo-persistent -o=go-template='URI value: {{.spec.source.git.uri | printf "%s\n"}}'
 ```
 
-Navigate to **"pelorus / Software Delivery Performance - By App"** and select the **todolist** application.
+#### View the Pelorus measurements
 
-You should now see at least one measurement for **Lead Time for Change** and **Deployment Frequency**, like in the following image.
+After a couple of minutes you should see at least one measurement for **Lead Time for Change** and **Deployment Frequency** in the Grafana dasboard, like in the following image:
+
+> **Note:** If the time elapsed from your `mongo-persistent` deployment is higher then **5 minutes**, you may want to adjust different relative time range <span style="color:red">⓵</span> e.g. **Last 3 hours**s.
 
 ![gnome-shell-screenshot-3zh4l2](../img/initial_measurement.png)
 
-### Update application
+#### Update application
 
-In this section we will
+In this section we will observe application continous delivery and analyze it with Pelorus. Few steps will guide us through this process:
 
-- Make changes to the application
-- Commit changes to source control
-- Watch the application redeploy with the changes to be captured by Pelorus
+- [Setting up GitHub Webhook](#github-webhook) that allows to automatically run new builds of your smaple applications when the commit to the source code happens
+- Fixing an issue in the running application and [update the application source code](#update-the-application-source-code)
+- Committing changes to source control
+- Watching the application redeploy with the changes to be captured by Pelorus
 
-#### Github Webhook
+##### Github Webhook
 
 You can easily watch how Pelorus works by automatically building and deploying the todolist app when a commit is pushed to Github by utilizing Github's webhooks.
 
 To get the build webhook URL you can navigate to the todolist BuildConfig details or by running
+```shell
+# Get the secret that is used by the Webhook
+$ WEBHOOK_SECRET=$(oc get bc/todolist -o=jsonpath='{.spec.triggers..github.secret}' -n mongo-persistent)
```

Comment:
two dots between triggers and github?

Reply:
Yes. The `.spec.triggers` is a list of dicts that contains different secrets so we need to use two dots:

```
[{"github":{"secret":"<secret>"},"type":"GitHub"},{"generic":{"secret":"<secret>"},"type":"Generic"},{"type":"ConfigChange"}]
```